docs/specifications/tests/Methods.md

```diff
@@ -1,69 +1,604 @@
-# Methods common to Test Specifications
+# Methods common to Test Case Specifications
 
-This is a list of generic methods used in many test specifications. The
-test cases that makes use of any of these methods should link directly to
+This is a list of generic methods used in many Test Case specifications. The
+Test Cases that makes use of any of these methods refer directly to
 this text.
 
-### Method 1: Obtain the parent domain
+## Method 1: Parent zone
 
-This metod tries to obain the parent domain from the domain used as input
-to this method.
+The method has been removed. The function is integrated in [BASIC01] instead.
 
-1. A recursive SOA-record lookup for the child domain name starting at the
-   root domain should be done, and the steps of the process recorded.
-2. If the recursion reaches a name server that responds with a redirect
-   directly to the requested domain, including functional glue, the test
-   succeeds. The domain through which the name server was found is
-   considered the parent domain.  
-3. If the recursion reaches a name server that authoritatively responds
-   with NXDOMAIN for the child domain, the test succeeds. The domain through
-   which the name server was found is considered the parent domain.
+-------------------------------------------------------------
 
-### Method 2: Obtain "glue Name records" from parent
+## Method: Get delegation NS names and IP addresses
 
-This method tries to obtain the authoritative name servers from the
-delegation of the parent domain.
+### Method identifier
+**get-del-ns-names-and-ips**
 
-1. Obtain the parent domain as input from Method 1.
-2. Send a query to the parent domain asking for the list of name servers
-   authoritative for the domain that is being tested 
-3. Record the list of name servers obtained from the authority section 
+### Objective
 
-### Method 3: Obtain name servers from child
+Obtain the name servers names (extracted from the NS records) and IP 
+addresses from any glue records for [in-bailiwick] name server names
+from the delegation of the given zone (child zone) from the parent 
+zone. Obtain the IP addresses for any [out-of-bailiwick] name server 
+names.
 
-Just as in Method 2, this method tries to obtain the name servers configured
-for the child zone from the child domain itself.
+This is an external method to be used by test cases.
```

Comment:
I don't see the point of declaring this. Technically there is no difference between external and internal methods. It's all fine and dandy that we thought of them that way when we dreamed them up, but I don't see a situation where this information is useful. In other words I think these declarations should be removed.

Reply:
It puts an unnecessary burden on the reader to decide whether this information is relevant to him or not, when we know beforehand that it isn't. Also, the more irrelevant information a reader is presented with, the more he'll want to skip over things that are probably not relevant anyway.

Reply:
We have different opinions on whether this is irrelevant or not. For a `human program` I find it relevant. Let us solve other issues first, and then we can see at the end.